docs/namespaces.md

```diff
@@ -0,0 +1,39 @@
+# Namespaces
+
+Chronicle namespaces allow you to separate chronicle data, avoiding collisions
+between identities and avoiding returning unnecessary results with your queries.
+
+A namespace consists of a label part and a UUID. If a namespace label does not
+exist, then using it will create a new namespace with that label and a random
+UUID.
+
+This can mean that multiple instances of chronicle can potentially have the same
+namespace _label_, but are in fact referring to different namespaces. We provide
+a namespace binding section in the [configuration](/config) file for this.
```

Comment:
```suggestion
a namespace binding section in the [configuration](config.toml) file for this.
```

Reply:
This is meant to be a doc link

Reply:
Ohh, sorry, expected an `.md` for those, should have guessed.